src/tdd/isograma.ts

```diff
@@ -1,9 +1,28 @@
-function isograma(string) {
-
-        if (string !== '-') {
-          callback(string);
-        }}
-
-
-
-export default isograma;
\ No newline at end of file
+function isIsograma(s: string): Array<string> {
+    let verdadeiroBo: string[] = [" ", "-", "lumberjacksL", "background", "downstream", "six-year-old"]
+    const resultado = s.replaceAll("-", " ");
+    
+    if (verdadeiroBo.includes("lumberjacksL")){
+        console.log("is true")
+    }
+    if (verdadeiroBo.includes("background")){
+        console.log("is true")
+    }
+    if (verdadeiroBo.includes("downstream")){
+        console.log("is true")
+    }
+    if (verdadeiroBo.includes("six-year-old")){
+        console.log("is true")
+    }
+    if (verdadeiroBo.includes(" ")){
+        console.log("is true")
+    }
+    if (verdadeiroBo.includes("-")){
+        console.log("is true")
+    }
+    for (let i = 0; i < isIsograma.length; i++) {
+        verdadeiroBo.push(s[i]);
+    }
```

Comment:
Esse for loop aqui está meio estranho, porque `isIsograma` é uma função, e está sendo usado como argumento no loop. Usar a função como condição de parada do loop (`; i < isIsograma.length;`) provavelmente não tem o comportamento esperado e não vai ajudar na resolução do problema.

Reply:
Fechado! Corrigido (removi) ele pois não fazia mais sentido.